File: rudolph/src/main/java/cn/wzbos/android/rudolph/ILogger.java
package cn.wzbos.android.rudolph;

interface ILogger {
    void v(String tag, String message);

    void d(String tag, String message);

    void i(String tag, String message);

    void w(String tag, String message);

    void e(String tag, String message);

    void e(String tag, Throwable throwable);

    void e(String tag, String message, Throwable throwable);
}

File: rudolph/src/main/java/cn/wzbos/android/rudolph/ILogger.kt
package cn.wzbos.android.rudolph

interface ILogger {
    fun v(tag: String?, message: String?)
    fun d(tag: String?, message: String?)
    fun i(tag: String?, message: String?)
    fun w(tag: String?, message: String?)
    fun e(tag: String?, message: String?)
    fun e(tag: String?, throwable: Throwable?)
    fun e(tag: String?, message: String?, throwable: Throwable?)
}